docs/resources/built-in-decorators.md

```diff
@@ -65,3 +65,32 @@ lint:
       tagNames:
         pet: ./my-custom-description.md
 ```
+
+### hide-internals
+
+Using `x-internal` tag by default to hide nodes:
+```
+  PathItem
+  Operation
+  Schema
+  Response
+  RequestBody
+  Example
+  MediaType
+  Server
+  Callback
+  Parameter
+```
+
+The `tagToHide` option uses to define name of tag.
```

Comment:
I don't understand this sentence or the example.

Reply:
Oh, this is the tag that is evaluated -- default is `x-internal`?

Reply:
I think given this the decorator name `hide-internals` could make sense (or `remove-internals`). However, this particular argument is unclear, because it doesn't just hide that tag, it hides the node/object it is included in such as an operation, property, etc...

Reply:
I think this should not have the word `tag` at all because the word has semantic meaning in OpenAPI.

Reply:
@adamaltman @RomanHotsiy, guys could you help me to define the name of this option please.

Reply:
I have a few suggestions:

- `propertyName`
- `byPropertyName` (it will have logical flow: remove-internals -> byPropertyName)
- `byProperty` (it will have logical flow: remove-internals -> byProperty)
- `internalPropertyName`

I don't have a strong opinion here. @adamaltman what do you think?

Reply:
I think @andriyl implemented `internalPropertyName` and it seems good.

Reply:
done